atomic_reactor/plugins/pre_fetch_sources.py

```diff
@@ -195,32 +199,44 @@ def _get_remote_urls_helper(self, koji_build):
         self.log.debug('archives: %s', archives)
         remote_sources_path = self.pathinfo.typedir(koji_build, btype='remote-sources')
         remote_sources_urls = []
+        dest_name = None
+        cachito_json_url = None
 
         for archive in archives:
             if archive['type_name'] == 'tar':
                 remote_source = {}
                 remote_source['url'] = os.path.join(remote_sources_path, archive['filename'])
                 remote_source['dest'] = '-'.join([koji_build['nvr'], archive['filename']])
+                dest_name = remote_source['dest']
                 remote_sources_urls.append(remote_source)
-        return remote_sources_urls
+
+            elif archive['type_name'] == 'json' and archive['filename'] == 'remote-source.json':
+                cachito_json_url = os.path.join(remote_sources_path, archive['filename'])
```

Comment:
well I want to be sure it is really there, hence in loop

Reply:
I see, but there does not seem to be any special behaviour for the case when it is missing. The URL will end up being `None` and it will eventually fail with `request_session.get(None)` when creating the full mapping? Is that OK?

Reply:
well remote-source.json will always be in source container build as well as just 1 archive

so I don't think we need extra check for that

in the worst case it will fail on that get(None) (IF source container build is missing remote-source.json)

Reply:
ok